react-native-in-app-payments-quickstart/README.md

```diff
@@ -111,3 +111,6 @@ Square provides a sample backend service for this quickstart that can easily be
 [root README]: ../README.md
```

Comment:
I think it would be better to add this into the `README.md` file at the root of the repo.

This repo houses a plugin/library for React Native into the IAP SDK. 

The `react-native-in-app-payments-quickstart` is just a sample app that lives within the repo.

Ideally to me this feedback link lives only in the top-level README

Reply:
changed!

Reply:
actually, the link specifically says "sample app", so maybe it doesn't make sense to have this in the root